Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml

```diff
@@ -30,10 +32,20 @@
                     <Ginger:ucTagsViewer x:Name="xTagsViewer" HorizontalAlignment="Left" Margin="-5,0,0,0"/>
                 </StackPanel>
 
-                <StackPanel Margin="0,5,0,0" >
-                    <Label Content="Page URL:" Style="{StaticResource $LabelStyle}"/>
-                    <TextBox x:Name="xURLTextBox" AutomationProperties.AutomationId="PageURL POMID" Style="{StaticResource $TextBoxStyle}" FontWeight="Bold"  />
+                <StackPanel Margin="0,0,0,0" HorizontalAlignment="Left" >
+                    <Label Margin="5,0,0,0" Style="{StaticResource $LabelStyle}" Content="Page Load Flow:" FontWeight="Bold" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                    <RadioButton x:Name="xRDBPageURL" Height="20" Content="Page URL"  Grid.Column="0" Grid.Row="1" Tag="radPageURL" Checked="XRDBPageURL_Checked" HorizontalAlignment="Left" Margin="10,5,0,0" />
```

Comment:
1- "xRDBPageURL" and "xRDBBF" are names which hard to understand their meaning on ode side, it should be "xPageUrlRadioBtn" and "xBusinessFlowRadioBtn"
2- you should have only single "Checked" event for both Radio btns which will handle the selected item on POM,

Reply:
renamed radio buttons, and the radiobutton is checked from code behind